test/test_getting_tables.py

```diff
@@ -444,3 +444,13 @@ def test_get_tables_with_leading_digits():
     assert ["0020_big_table"] == Parser(
         "SELECT t.val as value, count(*) FROM 0020_big_table"
     ).tables
+
+
+def test_insert_ignore_with_comments():
+    assert ["bar"] == Parser(
+        "INSERT IGNORE /* foo */ INTO bar VALUES (1, '123', '2017-01-01');"
+    ).tables
+
+    assert ["bar"] == Parser(
+        "/* foo */ INSERT IGNORE INTO bar VALUES (1, '123', '2017-01-01');"
+    ).tables
```

Comment:
I would also add inline comments at the beginning (with a new line at the end of comment), both comment style with `# ` and `-- `.

And maybe check one or two other query types than Insert? 🤔

Reply:
Good point, more tests added.